main.go

```diff
@@ -103,6 +103,7 @@ func mainWithError() (err error) {
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
 	daemonCommand.PersistentFlags().String(f.Service.Chart.Namespace, "giantswarm", "The namespace where chart CRs are located.")
+	daemonCommand.PersistentFlags().String(f.Service.Helm.TillerNamespace, "giantswarm", "Namespace for the Tiller pod.")
```

Comment:
It wasn't introduced until now.

Reply:
Cool makes sense. So far only tiller in tenant clusters was managed and its always in the giantswarm namespace.